scripts/classes/LocalSaving.gd

```diff
@@ -0,0 +1,28 @@
+extends Object
+class_name LocalSaveSystem
+
+const DEFAULT_PATH:String = "user://SaveFile.ini"
```

Comment:
Initially I would have said that I think this file should be placed in the "data" directory, the same as the json database does. But I am not convinced anymore that the data directory is the correct place. Maybe we should create a "user" directory to store all user files which should not be uploaded to git and place a .gitignore file in that directory with the following content:

``` 
# Ignore everything in this directory
*
# Except this file
!.gitignore
```

Reply:
Currently it is saved to the designated AppData folder that Godot handles, do you want it in the same folder as the game?